include/zephyr/bluetooth/mesh/blob_cli.h

```diff
@@ -419,6 +420,13 @@ uint8_t bt_mesh_blob_cli_xfer_progress_active_get(struct bt_mesh_blob_cli *cli);
  */
 bool bt_mesh_blob_cli_is_busy(struct bt_mesh_blob_cli *cli);
 
+/** @brief Set chunk sending interval in ms
+ *
+ *  @param cli BLOB Transfer Client instance.
+ *  @param interval_ms the delay after each chunk in ms.
+ */
+void bt_mesh_blob_cli_set_chunk_interval_ms(struct bt_mesh_blob_cli *cli, uint32_t interval_ms);
```

Comment:
What does it happen if I add delay for Pull mode? Will it break timing requirements?

Reply:
Will take a closer look. 
FYI How I tested my changes was mainly modifying the default value locally and running mesh bsim tests with reasonable interval times.

Reply:
I guess it is better to not recommend using it for pull mode at all since we cannot guarantee result. 1300ms was measured in some laboratory conditions that might be different in the deployed network in `the fields`.